coveo-systools/README.md

```diff
@@ -0,0 +1,13 @@
+# coveo-systools
+
+Language and OS related utilities.
+
+
+Content in a nutshell:
+
+- enhanced subprocess calls
+- file and app finding made easy
+- safe text write and replace-if-different
+- git-repo-root locator
+- bool platforms `if WINDOWS or LINUX or MAC or WSL:`
+- launch python modules (i.e.: python -m module) inside the current python process from python

```

Comment:
It would be _a lot_ better if you added a tiny example to each of these bullet points. I understand it's a bit tedious to do, but I think the improvement would be important. A lot of people code by "copy/pasting" examples.

Reply:
[x] examples for coveo-systools